i18n/i18n.go

```diff
@@ -70,21 +132,39 @@ func loadAsset(assetName string) {
 		panic(fmt.Sprintf("Could not load asset '%s': %s", assetName, err.Error()))
 	}
 
-	err = i18n.ParseTranslationFileBytes(assetName, bytes)
-	if err != nil {
+	if _, err := bundle.ParseMessageFileBytes(bytes, assetName); err != nil {
```

Comment:
Looks to be defining a second "err" variable in the same context. This would give inconsistent results.

Reply:
Can you elaborate? This method can return two different errors and I think we should panic with the correct error.

Reply:
Line 135 is creating a variable named `err`, but this variable was already created on line 130. Golang will let you create 2 variables with the same name in a scope, but the results you get from this can be unpredictable.

Reply:
Oh I see.  Yeah let me fix that